### PR TITLE
Xavier's second reflection

### PR DESCRIPTION
**Pull request by wxs**

### CODE REVIEW

File: content/post/wxs-reflection-002.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+These two kinds of time are in a sort of tension with one another. Chronos has this quality of practical necessity: people with a poor handle on Chronos tend to be chronically[^1] late, have trouble delivering complex projects. Kairos of a humane and cosmic empathy. People with a poor handle on Kairos tend towards burnout, for themselves and their collaborators, tend to be rigid in their thinking and planning having trouble adapting to changing circumstances, and may not respond well to the information held in intuitions and the natural ebb and flow of their energy.
```
Comment (sereprz) [inline content/post/wxs-reflection-002.md:11]: I went down a rabbit hole to find words that derive from Kairos in Italian (the Chronos ones are very obvious), and in doing so I came across [a third greek deity related to time](https://en.wikipedia.org/wiki/Aion_(deity)) -  I believe Aion is associated with a definition of time that's unbound (such a good word!) and cyclical - this idea of a returning version of the past, and I am curious of how this new "god" would fit into this thinking (if at all)

File: content/post/wxs-reflection-002.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+So, too, it seems, the fellowship should play skillfully with these materials. I have [an are.na board](https://www.are.na/xavier-snelgrove/time-material)where I've been collecting images and writings on the materiality of time which I should continue to contemplate on as part of my work. Some questions that come up here:
+- How rigid should the temporal boundaries of the fellowship container be?
+- How early does one decide to be in the fellowship? How does the container reach out to fellows in that earlier time?
+- How rigid is the scheduling inside the container?
+- How crisp should be the boundary of "graduation" or "completion"?
+- How does the container reach out to alumni into the future?
+
+One might think of the Chronos cycle a fellow goes through as something like:
```
Comment (sereprz) [inline content/post/wxs-reflection-002.md:22]: There is something elegant about the alternating of "discrete" events and "zones" here - almost as if each event opens up onto a new prospective or a flow new, different energy

File: content/post/wxs-reflection-002.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+There seems to be something powerful in being able to hold both of these times in balance. I'm struck with how this is seen in the language of the Catholic Church at its heyday, where there was a distinction made between its *temporal* and *atemporal* power; The temporal power of the church (read Chronos) was where it exerted control over mundane aspects of life, political and secular institutions. The atemporal power of the church (read Kairos) was its spiritual dimensions, understood to be somehow outside of the linear flow of time.
+
+So, too, it seems, the fellowship should play skillfully with these materials. I have [an are.na board](https://www.are.na/xavier-snelgrove/time-material)where I've been collecting images and writings on the materiality of time which I should continue to contemplate on as part of my work. Some questions that come up here:
```
Comment (mveytsman) [inline content/post/wxs-reflection-002.md:15]:
```suggestion
So, too, it seems, the fellowship should play skillfully with these materials. I have [an are.na board](https://www.are.na/xavier-snelgrove/time-material) where I've been collecting images and writings on the materiality of time which I should continue to contemplate on as part of my work. Some questions that come up here:
```
there's a missing space here

**Review by mveytsman — APPROVED** [review]
In my response to your reflection I wish to reflect on two things

## The distinction between people with a poor handle on chronos and people with a poor handle on kairos 

This binary personality classification makes me think of the ultimate binary people classifier: [round vs pointy](https://medium.com/the-hairpin/are-you-a-round-or-a-pointy-f4d5b1b2dd64) which delightful and I find myself coming back to every few years. 

Chronos is of course pointy and kairos is round. Although the analogy is broken when you add in desire:

> Pointies quite often wish that they were Rounds. Rounds never want to be anything but themselves. Pointies adore this game. Rounds find it quite tedious.

People with a poor sense of chronos, and I'm speaking of myself here, often really wish they didn't. I am not sure if a poor sense of kairos brings any peace at all.

## the kairal measure of the fellowship.

You laid out an eleven-stage cycle of the fellowship in chronal time. Is there a similar kairal cycle? Or is the kairal cycle of the fellowship so tied to the immediate experience that it can't be captured in words?

File: content/post/wxs-reflection-002.md (hunk 1 above)
Comment (wxs) [reply content/post/wxs-reflection-002.md:11]: Ooh, I love this, had not come across Aion, but I do love cyclical time... 

There's this idea of thinking about "7 generations" as a way to connect to deeper time. I think it was in the book "sand talk" that I learned how that is grounded in the timespan of living memory, that is: if great grandparents are pretty much the limits of lifespan, then a great grandparent can remember their great grandparents, their grandparents, their parents, themselves, their children, their grandchildren, and their great grand children.

In the book as I recall he says his great grandmother used to call him "great grandfather" thereby explicitly connecting the two horizons given by the limits of mortality into one cycle...

**Comment by wxs** [comment]
> You laid out an eleven-stage cycle of the fellowship in chronal time. Is there a similar kairal cycle? Or is the kairal cycle of the fellowship so tied to the immediate experience that it can't be captured in words?

Hmm I feel like maybe my reply to Serena around the Aionic cycle is maybe a partial response...

But yeah I like leaving some part of it also less tied to the clock at all, something more like the state of mind of being in the researcher mode, where suddenly everything reveals itself as a potential object of curiosity and study and depth.

**Pull request merged**